docs/sdk/src/guides/async_backing_guide.rs

```diff
@@ -185,7 +185,7 @@
 //!         client.code_at(block_hash).ok().map(|c| ValidationCode::from(c).hash())
 //!     },
 //!     ..
-//!     authoring_duration: Duration::from_millis(1500),
```

Comment:
I think this one needs to be increased as well: https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/polkadot-parachain/src/service.rs#L835

Right @s0me0ne-unkn0wn ?

Reply:
Westend still has 2sec backing timeout, so we should bump it on Westend first, and then we may bump it there.

It also should be considered that this change would affect all the system parachains as they're using `polkadot-parachain` to run collators, so, although it *should* be safe to bump it, I'd try it on our collators first, just in case.

Reply:
I think it should be fine, AFAIK nobody is actually fully saturating the execution.

Reply:
Well, some runtimes do saturate by design (like HydraDX) so they may start having problems on Westend. On the other hand, I doubt if they're using `polkadot-parachain` to collate :thinking: 
Overall, yes, should be totally fine, but one cannot be too paranoid with a change affecting system parachains :slightly_smiling_face: